server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -294,25 +292,6 @@ public void processShouldNotSpoolMailWhenNotSavingToOutbox() throws Exception {
     }
 
     @Test
```

Comment:
you don't need to add any unit test? (yes I've seen that you did a great coverage in integration tests)

Reply:
With mocking?

Sorry, but it ties down to much with lower layers, I really don't like it.

If you ask me for it, I prefer porting these tests to rely on InMemoryMailboxManager through InMemoryIntegrationResousources.

Reply:
That was just a question. It's not really needed as you are doing all these integration tests. Sometime you need to add some unit tests (so yes with mocking) to check special behaviour such as storage returning some exceptions.

Reply:
I have coverage for the error management I introduced, and BTW extended it ;-)

I don't think we end up in that scenario.